lib/srv/exec.go

```diff
@@ -363,8 +363,12 @@ func collectStatus(cmd *exec.Cmd, err error) (*execResult, error) {
 // but for unit testing it takes any file
 //
 // Returns a strings which looks like "PATH=/usr/bin:/bin"
-func getDefaultEnvPath(loginDefsPath string) string {
+func getDefaultEnvPath(osUser string, loginDefsPath string) string {
 	defaultValue := "PATH=" + defaultPath
+	variableName := "ENV_PATH"
+	if osUser == "root" {
```

Comment:
if `root` logs in without `ENV_SUPATH` defined, it won't pick up any path at all. instead, it should fall back to `ENV_PATH` in such case.

another question I have: is comparing username to `root` enough if a user is superuser? IMO a better test would be to check UID for 0.